velox/docs/develop/testing/fuzzer.rst

```diff
@@ -116,12 +116,49 @@ Fuzzers support a number of powerful command line arguments.
 
 * ``–-batch_size``: The size of input vectors to generate. Default is 100.
 
-In addition, Aggregation Fuzzer supports:
+There are also arguments that toggle certain fuzzer features:
+
+* ``--retry_with_try``: Retry failed expressions by wrapping it using a try() statement. Default is false.
+
+* ``--disable_constant_folding``: Disable constant-folding in the common evaluation path. Default is false.
```

Comment:
I think we don't want to advertise this flag. Ideally all fuzzer test should not disable constant folding. We use this flag only in XStream fuzzer test right now because we had an issue where common path constant folded the entire expression while simplified path evaluate the expression and throws.